sql/wsrep_high_priority_service.cc

```diff
@@ -416,6 +427,9 @@ int Wsrep_high_priority_service::apply_toi(const wsrep::ws_meta &ws_meta,
                                            wsrep::mutable_buffer &err) {
   DBUG_TRACE;
   THD *thd = m_thd;
+
+  ulonglong option_bin_log_save = thd->variables.option_bits & OPTION_BIN_LOG;
+
```

Comment:
I guess this is not necessay as `wsrep_apply_events()` may have handled this in a higher level. Can you check it?

Reply:
fixed and added detailed explanation what happens here